packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md

```diff
@@ -165,61 +109,34 @@ This resource contains detailed reference material on event types triggered with
-| IPAddress              | IP address                |       |         |
-
-### policy.entity_risk.evaluate
+### policy.continuous_access.evaluate
 
-**Description:** This event triggers when Okta receives a risk event and then evaluates the entity risk policy.
+**Description:** This event triggers when Okta detects a risk change event, and then evaluates the entity risk policy.
 
 | Key event properties | Description                                         | Data type      | Example values |
 | --------------------- | --------------------------------------------------- | -------------- | -------------- |
 | **event.system.debugContext.debugData**                |                 |         |         |
 | Behaviors             | List of behaviors identified for the current event. `POSITIVE` - the specific behavior is identified. `NEGATIVE` - the specific behavior wasn't identified. See [About Behavior Detection](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-behavior-detection).                 | key-value pairs        |  `{New Geo-Location=POSITIVE, New Device=NEGATIVE, New IP=POSITIVE, New State=POSITIVE, New Country=POSITIVE, Velocity=POSITIVE, New City=POSITIVE} `       |
-| Risk            | Contains the level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                   | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
+| Risk            | Contains the level of risk for a particular request (`LOW`, `MEDIUM`, or `HIGH`) and the `reasons` that contributed to the risk level.                | key-value pairs         |`{reasons=Anomalous Geo-Distance, New Device, New ASN, New IP, New State, New Country, New City, level=HIGH}`          |
 | ThreatSuspected            | If ThreatInsight is running and detects a request as suspicious, the value for this property is `true`.                   | Boolean         | `false`         |
 | TraceId            | A unique ID that is used across a single flow of ITP events to easily correlate them all into one System Log query                 | String         | `65d55fa6-b5a9-40f9-a6f1-627b9fa71b50`        |
-| **target** (User)         | The user associated with the risk change            | Object      |        |
+| **target** (User)         | The user associated with the risk change           | Object     |        |
 | type        | The type of target object     | String     | User       |
-| **target** (Policy)         | The entity risk policy           | Object     |        |
-| type        | The type of target object     | String     | Policy      |
-| **target** (Rule)         | The rule of the entity risk policy           |      |        |
+| **target** (Rule)         | The rule associated with the continuous access evaluation            | Object     |        |
```

Comment:
Thanks, yeah, for 2024.06.0 we're not updating dev docs, but it's on the to-do list. I'll keep things the same for this pr. It will get updated. Thanks